### DISCUSSION TITLE
bot可否实现米游社自动签到呢

### DISCUSSION THREAD

**Discussion by ID114514** [Arondight/Adachi-BOT#330] [Q&A] [answered]
cookie自动签到功能

**Comment by mark9804** [comment]
米游社自动签到需要拿到用户的cookie，然而大部分用户做不到这一步

退一步说，就算用户可以轻松提供自己的cookie，一个bot在任何时候也都不应该要求用户提供他的cookie。因为cookie当中记录了你的登录状态，是非常重要且秘密的数据，bot要求用户提供这种数据非常离谱。

如果你确保自己理解了把cookie交给未经过审计的程序的风险，并且仍然想使用类似的功能，可以前往：

https://github.com/y1ndan/genshinhelper2

**Reply by ID114514** [reply]
好的，谢谢

**Comment by Arondight** [comment, marked as answer]
我不会做收集玩家 cookie 的功能

**Reply by ID114514** [reply]
他们简直太疯狂了，为了自动签到甚至愿意把账号密码交给bot。

**Reply by Arondight** [reply]
> 直太疯狂了

算不上疯狂，我也挂 cookie 签到的，但是一旦有了什么财产纠纷，或者被资本制裁，我作为项目的开发者也没法全身而退
